lib/schema_registry/client.rb

```diff
@@ -27,11 +28,21 @@ def initialize(code, message)
 
   class Client
 
-    attr_reader :endpoint, :username, :password
+    attr_reader :endpoint, :username, :password, :client_key, :client_certificate, :ca_certificate, :verify_mode
 
-    def initialize(endpoint, username = nil, password = nil)
+    def initialize(endpoint, username = nil, password = nil, ssl_options = {})
```

Comment:
Can you use keyword arguments here, rather than a hash?

Reply:
@wvanbergen  mind if I ask why ?

Reply:
done